tests/aws/services/stepfunctions/v2/test_sfn_api.py

```diff
@@ -653,3 +653,35 @@ def test_create_update_state_machine_same_parameters(
             stateMachineArn=state_machine_arn
         )
         sfn_snapshot.match("describe_resp_t2", describe_resp_t2)
+
+    @markers.aws.unknown
```

Comment:
Why add an `unknown` marker here? Generally these markers should never really be added, they're just here for completeness sake. :thinking: